src/bot/root_pythia_cogs.py

```diff
@@ -63,7 +63,30 @@ async def resume(self, ctx):
             "Resumed successfully from idle state, requests can be sent again."
         )
 
-    # TODO: add a add_users command that would accept a list of ids
+    @commands.before_invoke(log_command_call)
+    @commands.command(name="addusers")
+    async def addusers(self, ctx, *args):
+        self.logger.info(f"command `addusers` received: {len(args)} arguments:")
+        for i in range(len(args)):
```

Comment:
a better variable name than `i` maybe? :smiling_face_with_tear:

Reply:
See commit https://github.com/iScsc/RootPythia/pull/27/commits/115ad9d4bf9043fdbc49409c18fbcdce50c89a56 for the fix ;)

Reply:
tks ❤️